### PR TITLE
Sdk Integration and Reset-run

### PR DESCRIPTION
**Pull request by Vivek205**
1. SDK integration in
    * CNTK Language Understanding
    * Face Alignment service
2. Reset and Run
3. Loader on service execution

### CODE REVIEW

File: src/assets/thirdPartyServices/snet/zeta36_chess_alpha_zero/alpha_zero_pb.js [hunk 1]
```diff
@@ -0,0 +1,440 @@
...
+ */
+// GENERATED CODE -- DO NOT EDIT!
+
+var jspb = require('google-protobuf');
+var goog = jspb;
+var global = Function('return this')();
+
+goog.exportSymbol('snet_zeta36_chess_alpha_zero.Input', null, global);
+goog.exportSymbol('snet_zeta36_chess_alpha_zero.Output', null, global);
+
```
Comment (chetankothari) [inline src/assets/thirdPartyServices/snet/zeta36_chess_alpha_zero/alpha_zero_pb.js]: We don't need `eslint-disable` here?

File: src/components/ServiceDetails/AboutService/ServiceDemo/ThirdPartyServiceErrorBoundary.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+  static getDerivedStateFromError = error => {
+    return { error };
+  };
+
+  componentDidCatch = (error, info) => {
+    this.setState({ error, info });
+  };
+
+  render() {
```
Comment (chetankothari) [inline src/components/ServiceDetails/AboutService/ServiceDemo/ThirdPartyServiceErrorBoundary.js]: Shall we get the design team to give us a nice UI for this?

Comment (Vivek205) [reply src/components/ServiceDetails/AboutService/ServiceDemo/ThirdPartyServiceErrorBoundary.js]: Sure.. will raise this concern with deesign team

File: src/assets/thirdPartyServices/snet/zeta36_chess_alpha_zero/alpha_zero_pb.js (hunk 1 above)
Comment (Vivek205) [reply src/assets/thirdPartyServices/snet/zeta36_chess_alpha_zero/alpha_zero_pb.js]: fixed in [commit](https://github.com/singnet/refactored-dapp/pull/122/commits/9a3e85577d17f47d863793d9a9207f15d36af266)

**Pull request merged**